app/src/main/java/nl/hogeschoolrotterdam/projectb/MemoryEditActivity.java

```diff
@@ -214,6 +216,14 @@ public void onClick(View v) {
                 startActivityForResult(i, LOCATION_EDIT);
             }
         });
+        spinner = findViewById(R.id.Marker_selector);
```

Comment:
this `findViewById` should be with the others (near line `77`). Otherwise it'll be harder to find.

Reply:
done